futures-core/src/executor.rs

```diff
@@ -1,32 +1,56 @@
-//! dox
+//! Executors.
 
 if_std! {
     use std::boxed::Box;
     use Future;
 
-    /// TODO: dox
+    /// A task executor.
+    ///
+    /// A *task* is a `()`-producing future that runs at the top level, and will
```

Comment:
Might be worth mentioning that Tasks are also the unit at which wakeups occur-- I get questions about this a lot.